files/image_config/interfaces/interfaces.j2

```diff
@@ -94,6 +94,11 @@ iface {{ name }} {{ 'inet' if prefix | ipv4 else 'inet6' }} static
 {% for route in MGMT_INTERFACE[(name, prefix)]['forced_mgmt_routes'] %}
     pre-down ip rule delete pref 32764 to {{ route }} table {{ vrf_table }}
 {% endfor %}
+{% if prefix | ipv6 and vrf_table == 'default'%}
+    # IPV6 default table not add to lookup by default, management server need this to access IPV6 address when BGP shutdown
+    up ip -6 rule add pref 32767 lookup {{ vrf_table }}
```

Comment:
>32767 [](http://example.com/codeflow?start=27&length=5)

How do you get the magic number?

Reply:
The number is based on IPV4 default route table priority:

admin@vlab-01:~$ ip -4 rule list
1001: from all lookup local
32764: from all to 172.17.0.1/24 lookup default
32765: from 10.250.0.101 lookup default
32766: from all lookup main
**32767**: from all lookup default

The default route table need have lowest priority so when can' find route from 'local' and 'main' table the rules from 'default' table will be use.

Reply:
Seems we need to define a named constant for 32764, and +3 here.